index.js

```diff
@@ -210,6 +211,30 @@ module.exports = function(version, _options) {
             }
 
             return output;
+        },
+        findAppropriateLanguage: function(language) {
```

Comment:
Does this method need to be exported? Also, how about calling it `getAvailableLanguage()` or `getBestMatchingLanguage()`?